.github/workflows/rust.yml

```diff
@@ -10,80 +15,79 @@ jobs:
       matrix:
         rust:
           - stable
-          - 1.44.0
+          - 1.63.0
```

Comment:
Just out of curiosity, was the MSRV bump required, or is this just to be consistent with other crates?

Reply:
I left a bit of justification in the commit message, I was seeing odd cargo download errors w/ 1.44: https://github.com/rusticata/rusticata-macros/pull/8/commits/f89acdfef33f887b142077c6506e0e7f6b00123f

We could try to find an alternative fix (or a lower version) but being consistent about the MSRV seemed useful generally (for a crate like this especially - I don't think it will see much use outside of the "parent" crates that already have a more strict MSRV).

Reply:
I also prefer to have the same version as much as possible.
rust 1.63 is from august 2022, and is present in debian bookworm. It is a good candidate